lib_xua/api/packet_sizes.h

```diff
@@ -0,0 +1,86 @@
+// Copyright 2017-2025 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+#ifndef _PACKET_SIZES_H_
```

Comment:
should this file be in /api? We dont want the user to include this directly right?